client/blocks/upload-image/index.jsx

```diff
@@ -250,7 +255,12 @@ class UploadImage extends Component {
 	}
 
 	removeUploadedImage = () => {
+		const { onUploadedImageRemove } = this.props;
+		const { uploadedImage } = this.state;
+
 		this.setState( { uploadedImage: null } );
```

Comment:
Any expected side-effects from leaving `selectedImage`, `selectedImageName`, and `editedImage` as-is?